hw/xfree86/dri/xf86dri.c

```diff
@@ -447,6 +442,8 @@ ProcXF86DRIGetDeviceInfo(register ClientPtr client)
         return BadValue;
     }
 
+    xXF86DRIGetDeviceInfoReply rep = { 0 };
+
     if (!DRIGetDeviceInfo(screenInfo.screens[stuff->screen],
                           &hFrameBuffer,
                           (int *) &rep.framebufferOrigin,
```

Comment:
Hmm, this cast in the old code is suspicious. Are we sure that `rep.framebufferOrigin` is of type `int` on every platform? We should check the same for other fields.